CoastalResilienceNetwork/GeositeFramework
# Update JS bundle

* Add more components and libraries used by Geosite Framework
plugins to the JS bundle.
* Update JS requirements file to include new dependencies.

This reduces the number of requests for JS resources from arcgis.com from 155 to 6 for the gulfmex region site. This site includes almost all of the Geosite Framework plugins.

**Testing Instructions**:

- Build a local version of the Gulfmex Region site.
- Update the JS bundle used there to version 1.0.8.
- Verify that there are a reduced number of requests for JS resources from arcgis.com

Connects to #468
